layers/README.md

```diff
@@ -23,10 +23,12 @@ The Layer class provides format validation and read/write capabilities to aid in
 
 | method [x = Layer()]| description |
 |:-------|:------------|
-| x.load_input(_input_) | Loads an ATT&CK layer from either a dictionary or a string representation of a dictionary. |
-| x.load_file(_input_) | Loads an ATT&CK layer from a file location specified by the _input_. |
-| x.export_file(_input_) | Saves the current state of the loaded ATT&CK layer to a json file denoted by the _input_. |
-| x.get_dict() | Returns a representation of the current ATT&CK layer object as a dictionary. | 
+| x.from_str(_input_) | Loads an ATT&CK layer from either a string representation of a dictionary. |
```

Comment:
> Loads an ATT&CK layer from either a string representation of a dictionary.

No longer accurate, it's just from a string now for this method.

Reply:
It's a string representation of a json object, to be completely precise.